### PR TITLE
XW-1387 | use htmlTitle parts from WikiVariables to build documentTitle

### PR DESCRIPTION
**Pull request by tomasznapieralski**
## Links

* http://wikia-inc.atlassian.net/browse/XW-1387

## Reviewers

@Wikia/x-wing

### CODE REVIEW

File: front/main/app/models/wiki/article.js [hunk 1]
```diff
@@ -189,6 +188,10 @@ ArticleModel.reopenClass({
 
 			// @todo this will be cleaned up in XW-1053
 			articleProperties.articleType = articleProperties.type || data.articleType;
+
+			articleProperties.documentTitle = articleProperties.isMainPage ?
+					'' :
+					articleProperties.documentTitle = articleProperties.displayTitle || articleProperties.title;
```
Comment (kamilkoterba) [inline front/main/app/models/wiki/article.js]: Add explanation that this is edge case for main page

**Comment by kamilkoterba** [comment]
except minor comment, LGTM

**Comment by kamilkoterba** [comment]
Thanks! docs with emphasis on why are always good
👍

**Comment by tomasznapieralski** [comment]
tested, works with old and new app version

**Pull request merged**